src/stores/connectors/vaults/create_aws.go

```diff
@@ -0,0 +1,25 @@
+package vaults
```

Comment:
Vaults are not connectors since they are not containing any business logic. They are wrappers around the infra clients so I would rather place them elsewhere, I would suggest `/src/stores/vaults` since it is the same kind of level of components of `/src/stores/store`

Reply:
moved to its own domain